--- host-automake-1.15.1/bin/aclocal.in.orig	2017-11-14 11:04:35.616062597 +0300
+++ host-automake-1.15.1/bin/aclocal.in	2017-11-14 11:07:30.513096684 +0300
@@ -69,7 +69,7 @@
 # ACLOCAL_PATH environment variable, and reset with the '--system-acdir'
 # option.
 my @user_includes = ();
-my @automake_includes = ("@datadir@/aclocal-$APIVERSION");
+my @automake_includes = ('@datadir@/aclocal-' . $APIVERSION);
 my @system_includes = ('@datadir@/aclocal');

 # Whether we should copy M4 file in $user_includes[0].
